Client/Frontend/Browser/ErrorPageHelper.swift

```diff
@@ -326,10 +328,18 @@ extension ErrorPageHelper: TabContentScript {
     }
 
     func userContentController(_ userContentController: WKUserContentController, didReceiveScriptMessage message: WKScriptMessage) {
+        guard let body = message.body as? [String: String] else {
```

Comment:
Can we have a utility method  in TabContentScript for the security token check, I believe this part is repeating in `didReceiveScriptMessage` for couple of Helpers

Reply:
@soner-yuksel - Using it here: https://github.com/brave/brave-ios/pull/3048/files#diff-baf1f99d0cf9134b5314430e98e858b4546f12abbc3fe5792b5b16939cddcc79R335

The way the body is parsed out of the message by different helpers is different, so I don't think adding `body = message.body` code to the helper function makes sense. Let me know if I'm missing something

Reply:
I meant this part is repeated with  `UserScriptManager.isMessageHandlerTokenMissing(in: body) ` check at least 3 time but it is not a big deal, maybe it is better this was for readability